views/components/Graphs/BarGraph.jsx

```diff
@@ -54,7 +54,7 @@ const BarGraph = ({ graph }) => {
           formatter={(value, name, props) => {
             const { payload } = props
 
-            return `${payload.counts[name]} (${value.toFixed(0)}%)`
+            return `${payload.counts[name]} (${value?.toFixed(0) || 0}%)`
```

Comment:
I agree with Will that we should extract the `value?.toFixed(0) || 0` logic to a helper function.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/187/commits/e579de9e94552417b0180ab7686f020c96a2462e